substrate-archive-backend/src/database.rs

```diff
@@ -104,7 +92,6 @@ impl ReadOnlyDB for SecondaryRocksDB {
         // need to make sure this is `Some` to open secondary instance
         let db_path = db_path.as_path().to_str().expect("Creating db path failed");
         let mut db_config = Config {
-            track_catchups: true,
```

Comment:
Is this a breaking change? Like, did we expose this option? If yes, it's changelog-worthy.

Reply:
Technically, no because `Config` is not re-exported but I think it should be re-exported because otherwise, it not possible to instantiate `SecondaryRocksDB` AFAIU.

Reply:
Yeah it was never directly exposed, only printed on shutdown